src/controllers/capp_test.go

```diff
@@ -0,0 +1,364 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+	cappv1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+	"github.com/dana-team/platform-backend/src/types"
+	"github.com/dana-team/platform-backend/src/utils/testutils"
+	"github.com/dana-team/platform-backend/src/utils/testutils/mocks"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"testing"
+)
+
+// createTestCapp creates a test Capp object.
+func createTestCapp(name, namespace string, labels, annotations map[string]string) {
+	cappRevision := mocks.PrepareCapp(name, namespace, labels, annotations)
+	err := dynClient.Create(context.TODO(), &cappRevision)
+	if err != nil {
+		panic(err)
+	}
+}
```

Comment:
This function is a copy of this one here:

https://github.com/dana-team/platform-backend/blob/9ee462faa5f5ecd9d3d3f20c6999a1c29ecf0f6f/src/routes/v1/capp_test.go#L20-L27